.github/workflows/release.yml

```diff
@@ -15,10 +15,11 @@ jobs:
     - uses: actions/checkout@v2
     
     - name: Set up Ruby
-      uses: actions/setup-ruby@v1.1.2
+      uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.6
-    
+        bundler-cache: true
```

Comment:
@mllemango been using [act](https://github.com/nektos/act) to try to test locally, but I think it's not working because I don't have the secrets. Do you know if there's a better way to test locally?

Reply:
I've always used act! Got it to run with some local changes so I think it runs without issue! Sorry one more note 😣 We can remove "install dependencies" now that we're using ruby/setup-ruby

Reply:
@mllemango removed Installed Dependencies. LMK if this is good to 🚢